sparkdl/__init__.py

```diff
@@ -12,3 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+# pylint: disable=wrong-import-position
+# pylint: disable=invalid-name
```

Comment:
Why do we need these given there is no import here? What is the invalid name?

Reply:
ditto